### ISSUE TITLE
Investigate what happens when mixing standard and partial updates

### ISSUE THREAD

**Issue by SarahFrench** [hashicorp/terraform-provider-google#17849]
### What kind of contribution is this issue about?

Other (specify in details)

### Details

This issue is blocked by issues changing the update verb for the given field: https://github.com/hashicorp/terraform-provider-google/issues/17832

Someone needs to investigate if the problem can be solved by changing to using 2x partial updates, or if a change to templates is required.

Also, possibly as separate ticket, alerting to identify this issue would be helpful to contributors.

### References

https://github.com/hashicorp/terraform-provider-google/issues/17832
